pdal/GDALUtils.hpp

```diff
@@ -848,7 +867,7 @@ class PDAL_DLL Raster
 
     GDALError wake();
 
-    std::string m_errorMsg;
+    mutable std::string m_errorMsg;
```

Comment:
Why did this change to mutable too?

Reply:
So you can set error state in const functions.